services/port_manager/src/main/java/com/futurewei/alcor/portmanager/repo/PortRepository.java

```diff
@@ -275,7 +275,7 @@ public synchronized void createPortBulk(List<PortEntity> portEntities, Map<Strin
                     .stream()
                     .collect(Collectors.toMap(PortEntity::getId, Function.identity()));
             portCache.putAll(portEntityMap);
-            neighborRepository.createNeighbors(portEntities.get(0).getProjectId(), neighbors);
+            //neighborRepository.createNeighbors(portEntities.get(0).getProjectId(), neighbors);
```

Comment:
Let us remove those commented lines when the change is confirmed. Also can we get rid of neighbor repository completely?

Reply:
Yes, will get rid of neighbor repository, NeighborProcessor.java and FetchPortNeighborRequest.java in next PR.

Reply:
Cool!